### PR TITLE
Add ElasticSearch machines

### PR DESCRIPTION
**Pull request by issyl0**

### CODE REVIEW

File: terraform/projects/app-elasticsearch/additional_policy.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "Statement": [
+        {
+            "Sid": "Stmt1499854881000",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:AttachVolume",
+                "ec2:DetachVolume",
+                "ec2:DescribeVolumeStatus",
+                "ec2:DescribeVolumes",
+                "ec2:DescribeInstances"
```
Comment (surminus) [inline terraform/projects/app-elasticsearch/additional_policy.json]: This is now a default policy, sorry!

File: terraform/projects/app-elasticsearch/main.tf [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+}
+
+variable "elasticsearch_2_subnet" {
+  type        = "string"
+  description = "Name of the subnet to place the  Elasticsearch 2 and EBS volume"
+}
+
+variable "elasticsearch_3_subnet" {
+  type        = "string"
+  description = "Name of the subnet to place the Graphite instance 3 and EBS volume"
```
Comment (surminus) [inline terraform/projects/app-elasticsearch/main.tf]: This is incorrectly described (probably in logs-elasticsearch, too)

File: terraform/projects/app-elasticsearch/additional_policy.json (hunk 1 above)
Comment (issyl0) [inline terraform/projects/app-elasticsearch/additional_policy.json]: Removed!

File: terraform/projects/app-elasticsearch/main.tf (hunk 2 above)
Comment (issyl0) [inline terraform/projects/app-elasticsearch/main.tf]: I think we need to go through everything - there must be so much that's mistakenly copy/pasted.

Comment (issyl0) [inline terraform/projects/app-elasticsearch/main.tf]: As an example: https://github.com/alphagov/govuk-aws/blob/d9db0d750bf194b8f2be2f3864889ab40eb41494/terraform/projects/app-elasticsearch/main.tf#L3, and that's also in `logs-elasticsearch` 'cause I copied that directory.

**Pull request merged**